### DISCUSSION TITLE
The future state of UI for Apache Kafka

### DISCUSSION THREAD

**Discussion by Haarolean** [provectus/kafka-ui#4255] [Announcements]
Dear community, 

I'd like to share an official statement from Provectus, the company behind UI for Apache Kafka:

-----
<i>Over the next few months, we'll be taking a pause in the active development of UI for Apache Kafka . This decision wasn't made lightly. Due to some internal strategic realignments and the need to assess our current priorities, we believe this pause is necessary to ensure that when we do proceed, we're doing so in the most effective and impactful way possible.

Please understand that this is not a termination of the project but a short hiatus. We're committed to UI for Apache Kafka and value the community that has grown around it.

During this pause:

The repository will remain accessible, and we encourage the community to continue exploring, using, and providing feedback on the existing codebase.
- We won't be merging new pull requests or actively addressing issues, but community discussion is still welcome.
- We'll remain reachable for any major concerns or critical issues that may arise.
- We look forward to the road ahead and appreciate your patience and understanding during this period. Once we have more clarity on the next steps and timelines, we promise to communicate them.

Thank you for your continued support and dedication.</i>

-----

Meanwhile, personally, I'm still committed to uphold the community in my free time, but as an **independent** contributor from now on.

A reminder, we also have a discord community, some activity happens there as well. Invite link: [here](https://discord.gg/4DWzD7pGE5)
The discussion of the same announcement on discord: [here](https://discord.com/channels/897805035122077716/897805085608935444/1154083205129969795)

**Comment by Haarolean** [comment]
The comments are on, but please keep the discussion civil :)

**Reply by stanyakimov** [reply]
Thanks for the update @Haarolean!

**Comment by nivhen** [comment]
One of the best Projects here, hope it will continue!

**Comment by muhib349** [comment]
Any upcoming plan, when would you restart your journey?

**Comment by james-newell-forge** [comment]
I hope you come back soon, do you think it would be possible to cut a new release in the mean time?

**Reply by neoXfire** [reply]
That would be usefull especially for features merged on master branch but not released yet (audit logs for example)

**Reply by Haarolean** [reply]
@james-newell-forge @neoXfire Please join us on discord to keep in touch 👀

**Reply by Haarolean** [reply]
An update to my comment: The new release could be found here: https://github.com/kafbat/kafka-ui

**Comment by fflatorre** [comment]
Hope this comment will reach Provectus since I don't know where else to write it : 
Can you please make the repo read-only and add a clear banner saying that the project is not anymore in active development? Is very confusing for people landing on the main page and not having a clear an honest report on the status of the project, maybe suggesting alternatives.

**Comment by IgorTsapko** [comment]
Just best UI ever. Thank you for all your job.

**Reply by Haarolean** [reply]
Thank you for the kind words!
You can find the new repo (and even a fresh release) here: https://github.com/kafbat/kafka-ui

**Reply by IgorTsapko** [reply]
Oh, really?? Cool! Just amazing! Happy that such gorgeous project hasn't passed away and has got the new life, thank you!

**Reply by Haarolean** [reply]
Thanks! Please leave a star and tell your friends/colleagues about us :) Any support is appreciated ❤️

**Comment by Haarolean** [comment]
A separate comment so it doesn't get lost under other comments branches:
The project now lives here: https://github.com/kafbat/kafka-ui
Happy to see your isssues and contributions there!

**Reply by larssb** [reply]
Mark this repo. as archived then plz. And thank you for the great work you've done over the years on the kafka-ui. It was a great tool as long as it lasted. Much appreciated.

Marking it as archived would make it easier for users to know what's up. And when marking it highlight - as done in your above comment - that kafbat/kafka-ui is the supported go-to.

**Reply by Galleleo** [reply]
This is becoming more dangerous btw: https://github.com/kafbat/kafka-ui/issues/1976
